Core/src/androidTest/java/com/paypal/android/core/HttpIT.kt

```diff
@@ -0,0 +1,37 @@
+package com.paypal.android.core
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+class HttpIT {
+
+    private lateinit var testDispatcher: TestCoroutineDispatcher
+
+    @Before
+    fun beforeEach() {
+        testDispatcher = TestCoroutineDispatcher()
+    }
+
+    @After
+    fun afterEach() {
+        testDispatcher.cleanupTestCoroutines()
+    }
+
+    @Test
+    fun itMakesAnHttpRequest() = runBlockingTest {
```

Comment:
Same comment here - do we want to establish a test naming pattern?

Reply:
Yeah that makes sense. I think leading with the method name makes sense. Went with `send_makesAnHttpRequest` here.